oneview_redfish_toolkit/multiple_oneview.py

```diff
@@ -176,8 +218,8 @@ def execute_query_ov_client(ov_client, resource, function, *args, **kwargs):
         try:
             result = ov_function(*args, **kwargs)
             return result
-        except Exception as e:
-            raise e
+        except Exception as exception:
```

Comment:
We can remove the `except` clause, we are just raising the exception catched.
The code at the `finally` clause will always be executed even without the `except` clause.